java/src/jmri/jmrix/can/CanSystemConnectionMemo.java

```diff
@@ -207,6 +208,19 @@ public void setSupportsCVHints(boolean b) {
         _supportsCVHints = b;
     }
     
+    /**
+     * Get the behaviour on ARST opcode
```

Comment:
I think it would be good if the the abbreviation ARST is explained here. It makes it easier for others to understand what this method does.